projects/kit/components/text-area/text-area.component.ts

```diff
@@ -139,6 +139,12 @@ export class TuiTextAreaComponent
         );
     }
 
+    get pseudoContent(): {fitted: string; extra: string} {
```

Comment:
We do not create objects in getters, it has an overhead which is bad for performance. Make 2 getters instead.

Reply:
i made 2 getters